README.md

```diff
@@ -2,13 +2,24 @@
 
 Library for creating and manipulating OcTrees as well as creating differential operators discretized on OcTree meshes.
 
-#Requirements
+# Requirements
 
-JOcTree is intended for use with Julia 0.4.x. The jInv is the only required dependency. 
+JOcTree is intended for use with Julia 0.4.x. Other dependencies are
```

Comment:
I was thinking of merging these changes, tagging a final 0.4 version and then making another PR for the 0.5 stuff. I think this makes for a cleaner history than me pulling in this stuff to my local copy and pushing everything at once but we could also do it that way.